tasks/main.yml

```diff
@@ -1,4 +1,9 @@
 ---
+- name: install python libselinux package
+  package: name=python3-libselinux state=latest
+  become: true
+  when: ansible_selinux is defined and ansible_selinux != False and ansible_selinux.status == 'enabled'
```

Comment:
IIUC, if we bump the minimum required version of Ansible, checking the value of `ansible_selinux` could be omitted: https://github.com/ansible/ansible/pull/31065.

Reply:
Oh, and https://github.com/abrt/ansible-abrt-client-role/pull/3 does bump the version.

Reply:
I removed the `ansible_selinux != False` check as you've suggested.